packages/unistyles/src/core/ServerUnistylesStyle.tsx

```
@@ -0,0 +1,58 @@
+
+export const isReactServerComponentRender = () => {
+    if (!isServer()) {
+        return false
+    }
+
+    if (serverReact.cacheSignal) {
+        return Boolean(serverReact.cacheSignal())
+    }
+
+    return Boolean(
+        getCachedServerComponentMarker && getCachedServerComponentMarker() === getCachedServerComponentMarker(),
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)